src/Tizen.Location/Tizen.Location/Locator.cs

```diff
@@ -352,13 +352,25 @@ public void Start()
         public void Stop()
         {
             Log.Info(Globals.LogTag, "Stopping Location Manager");
-            int ret = Interop.Locator.Stop(_handle);
-            if (((LocationError)ret != LocationError.None))
+            if (_batchPeriod > 0 && _batchPeriod > _batchInterval)
```

Comment:
extract method or variable
```suggestion
            bool isBatched = _batchPeriod > 0 && _batchPeriod > _batchInterval;
            if (isBatched) 
```

Reply:
You need to mark StartBatch was invoked, 
property check is not enough. 

``` c#
  locator.Start(); // it is just Start()
  locator.BatchPeriod =200;
  locator.BatchInterval = 10;
  locator.Stop(); // it shoud call Stop() but in your implement will call BatchStop()
```